### PR TITLE
ci: Remove dependabot config so that reviewers can be added in dependabot dashboard

### PR DESCRIPTION
**Pull request by rafenden**

### CODE REVIEW

File: .dependabot/config.yml [hunk 1]
```diff
@@ -11,4 +11,9 @@ update_configs:
     default_reviewers:
       - paulgain
       - rafenden
-      - yeahfro
+      - debitan
+      - ian-leggett
+      - paulgain
+      - peterhudec
+      - web-bert
+      - yozzo
```
Comment (web-bert) [inline .dependabot/config.yml]: @rafenden can this be the @uktrade/datahub-front-end  team?

Comment (paulgain) [reply .dependabot/config.yml]: @rafenden I'm on the list twice.

Comment (rafenden) [reply .dependabot/config.yml]: > @rafenden can this be the @uktrade/datahub-front-end team?
Good idea, will update the code.

Comment (rafenden) [reply .dependabot/config.yml]: Unfortunately, this is currently not supported 😢 
https://github.com/dependabot/feedback/issues/546

Comment (rafenden) [reply .dependabot/config.yml]: Unfortunately, this is currently not supported 😢
dependabot/feedback#546

**Pull request merged**